layers/meta-balena-genericx86/recipes-support/resin-init/resin-init-flasher-board/resin-init-flasher-board

```diff
@@ -57,6 +57,11 @@ sync
 
 [ -z $INSTALL_UEFI ] && exit 0
 
+# remove existing resinOS entries so we won't have duplicates
```

Comment:
balenaOS

Reply:
The entry is still named resinOS currently.

Reply:
I see.